### DISCUSSION TITLE
How to Change The Observation Name Displayed On Page

### DISCUSSION THREAD

**Discussion by gary-hammer** [bellrichm/weewx-jas#92] [Q&A]
My example is Apparent Temperature.
I'd like to have it display as Feels Like
I changed this under weewx.conf Labels but while it has effect on seasons skin there is no change in jas
I also changed the display name of Outdoor Humidity to Humidity and a couple of others.
Seems to be mighty long character-wise for a smaller display like my tablet.

**Comment by bellrichm** [comment]
Sounds like a bug. I’ll take a look.

**Reply by bellrichm** [reply]
According to, https://www.weewx.com/docs/customizing.htm#how_to_change_labels, the following should work
```
    [[jas]]
        skin = jas
        lang = en
        unit_system = US
        enable = true
        [[[Labels]]]
            [[[[Generic]]]]
                outTemp = Outside
```
Issue #95 has been created
